build-scripts/config_common.cmake

```
@@ -165,4 +165,7 @@ endif ()
 if (DEFINED WAMR_APP_THREAD_STACK_SIZE_MAX)
   add_definitions (-DAPP_THREAD_STACK_SIZE_MAX=${WAMR_APP_THREAD_STACK_SIZE_MAX})
 endif ()
+if (WAMR_BUILD_CUSTOM_NAME_SECTION EQUAL 1)
+  add_definitions (-DWASM_ENABLE_CUSTOM_NAME_SECTION=1)
```

Had better print a message to indicate the feature is enabled
```suggestion
  add_definitions (-DWASM_ENABLE_CUSTOM_NAME_SECTION=1)
  message ("     Custom name section enabled")
```